examples/Makefile

```diff
@@ -0,0 +1,11 @@
+WORK_DIR:=$(strip $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+SRC_DIR?=$(WORK_DIR)
```

Comment:
When would these be set by the developer?

Reply:
When you put your Makefile in a top level directory your proto files in another directory and your generated code in yet another. Sort of like setup.py doesn't live in the source directory. If it bugs you we can remove it.

Reply:
But this Makefile is only for the user protobuf file in this directory, so why would someone need to use alternate directories?